bootstrap_actions/emr-setup.sh

```diff
@@ -115,6 +115,10 @@ EOF
 
     /var/ci/update_dynamo.sh &
 
+    chmod u+x /var/ci/status_metrics.sh
```

Comment:
```suggestion
```

I think the script that downloads these files makes them executable already. I would remove tile 114 as well as it is not necessary either. 

this should capture your script as well I think 
https://github.com/dwp/aws-clive/blob/master/bootstrap_actions/download_scripts.sh#L37